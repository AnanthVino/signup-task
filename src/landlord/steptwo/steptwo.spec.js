import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import StepTwo from '.';
    
describe('Landlord StepTwo', () => {
  it('should render correctly', () => {
    
    const wrapper = shallow(<StepTwo />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})