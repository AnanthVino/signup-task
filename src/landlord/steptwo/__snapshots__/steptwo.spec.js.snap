// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landlord StepTwo should render correctly 1`] = `
<div
  className="container d-flex flex-column justify-content"
>
  <form>
    <div
      className="form-group"
    >
      <label>
        Password
      </label>
      <input
        className="form-control input-box rounded-0"
        onChange={[Function]}
        placeholder="Password"
        type="password"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Country
      </label>
      <select
        className="form-control input-box rounded-0"
        defaultValue=""
        onChange={[Function]}
      >
        <option
          disabled={true}
          value=""
        >
          Select
        </option>
        <option
          className="option"
          value="Kuwait"
        >
          Kuwait
        </option>
        <option
          className="option"
          value="Saudi Arabia"
        >
          Saudi Arabia
        </option>
        <option
          className="option"
          value="United Arab Emirates"
        >
          United Arab Emirates
        </option>
      </select>
    </div>
    <div
      className="form-group"
    >
      <label
        className="label"
      >
        Gender
      </label>
      <div
        className="p-t-10"
      >
        <label
          className="gender-container m-r-45"
        >
          Male
          <input
            checked="checked"
            name="gender"
            onChange={[Function]}
            type="radio"
            value="male"
          />
          <span
            className="checkmark"
          />
        </label>
        <label
          className="gender-container"
        >
          Female
          <input
            name="gender"
            onChange={[Function]}
            type="radio"
            value="female"
          />
          <span
            className="checkmark"
          />
        </label>
      </div>
    </div>
  </form>
  <div
    className="d-flex justify-content-between"
  >
    <Link
      className="btn btn-primary rounded-0"
      to="/landlord/stepone"
    >
      Previous Step
    </Link>
    <button
      className="btn btn-success rounded-0"
      onClick={[Function]}
    >
      Submit
    </button>
  </div>
</div>
`;
